source/common/upstream/resource_manager_impl.h

```diff
@@ -27,19 +27,22 @@ class ResourceManagerImpl : public ResourceManager {
 public:
   ResourceManagerImpl(Runtime::Loader& runtime, const std::string& runtime_key,
                       uint64_t max_connections, uint64_t max_pending_requests,
-                      uint64_t max_requests, uint64_t max_retries,
+                      uint64_t max_requests, uint64_t max_retries, uint64_t max_connection_pools,
```

Comment:
note: this change is going to conflict with https://github.com/envoyproxy/envoy/pull/6269. We should probably merge that one first? cc @spenceral

Reply:
Let me know when that has been done and I'll deal with the conflict.

Reply:
Looks like it's merged now. I'll bring it into my branch.